collectors/python.d.plugin/python.d.conf

```diff
@@ -26,7 +26,7 @@ gc_run: yes
 gc_interval: 300
 
 # apache: yes
-
+edgex: yes
```

Comment:
Given that this does not seem to be particularly lightweight and most users are (statistically speaking) not likely to be using it, I'm of the opinion that it should probably be disabled by default.

Reply:
> (statistically speaking)

I really liked the diplomacy here, it made me chuckle😋.  You are right, there is no sense to enable a plugin that will fail to gather metrics for the grand majority of use-cases.